scan/package.json

```diff
@@ -22,6 +22,7 @@
     "js-sha256": "^0.9.0",
     "moment": "^2.24.0",
     "react": "^16.8.0-0",
+    "react-highlight": "^0.12.0",
```

Comment:
Remove

Reply:
I think we should complete #214 first then rebase it to this PR